### DISCUSSION TITLE
part-ii/2018-121/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#205] [Question comments]
Part II 2018 Probability and Measure: Paper 2, Section II, J 

https://questions.tripos.org/part-ii/2018-121/

**Comment by eha1234** [comment]
<details><summary>(a)</summary>

Holder's inequality
</details>

<details><summary>(b)</summary>

Construct $n_k$ by taking $n_k$ to be sufficiently large that you have control over $\sum_{k<j} \mathbb{E}(Y_k Y_j)$, for instance, you can inductively define $n_k$ so that $| \mathbb{E}(X_{n_i}X_{n_j}| \leq \frac{1}{2^j(j-1)}$
</details>

<details><summary>(b)</summary>

$L^1$ convergence: $L^2$ bounded implies UI by Holder's inequality. The rest is bookwork. 

$L^2$ convergence: No. Take $X_n = n^{1/2} 1_{\{0<w < 1/n\}}$. Have $\mathbb{E}(X_n) = 1$ and it converges almost everywhere to 0 (hence converges in probability to 0), but it does not converge in $L^2$. 
</details>

<details><summary>(d)</summary>

Define $Z_n = X_n - \mathbb{E}(X_n)$, bounded in $L^2$. Calculate $||\frac{1}{n}\sum_{n=1}^N Z_n ||_2^2$ to see that it converges in $L^2$ to $0$. Now $\frac{1}{N}\sum X_n = \frac{1}{N} \sum Z_n + \frac{1}{N}\sum \mathbb{E}X_n$ so the condition is that   $\frac{1}{N}\sum \mathbb{E}X_n$ must converge 
</details>
